### PR TITLE
Updates release & adds Ubuntu 22.04

### PR DESCRIPTION
**Pull request by asachs01**
This PR updates the version of the plugin to use the latest upstream release (2.3.1 vs 2.2.0) and also adds Ubuntu 22.04 support.

### CODE REVIEW

**Comment by asachs01** [comment]
I've also validated that downloading the latest version and compiling seems to fix issues with `check_ping`

**Comment by jspaleta** [comment]
Okay this looks mergable.  But this plugin needs some love to transition from travis into github action based CI/CD before we can realistically do a new release.

**Comment by jspaleta** [comment]
I have to wonder... 
If the existing ubuntu build doesn't work on latest Ubuntu... something else is wrong.
We shouldn't need to rebuild for all possible linux distributions if we are doing the library detection correctly because of the way gcc symboling works.
In general we should eb able to use the oldest supported version of a particular distro family as the asset tarball and reference it in multiple builds using the appropriate filters in the asset resource definition.

File: Dockerfile.ubuntu2204 [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:22.04 as builder
+
+ARG SENSU_GO_ASSET_NAME="monitoring-plugins-ubuntu2004"
```
Comment (jspaleta) [inline Dockerfile.ubuntu2204:3]:
```suggestion
ARG SENSU_GO_ASSET_NAME="monitoring-plugins-ubuntu2204"
```

**Review by jspaleta — CHANGES_REQUESTED** [review]

**Pull request merged**